src/files-and-videos/videos-page/data/api.js

```diff
@@ -174,15 +174,17 @@ export async function uploadVideo(
   uploadFile,
   edxVideoId,
 ) {
-  const formData = new FormData();
-  formData.append('uploaded-file', uploadFile);
+  // const formData = new FormData();
```

Comment:
I don't understand this change. What is the purpose?

Reply:
I think this one was an accidental commit while bouncing between branches